tasks/task_get.go

```diff
@@ -0,0 +1,35 @@
+package tasks
+
+import (
+	"database/sql"
+	"encoding/json"
+	"net/http"
+
+	"github.com/kosttiik/todo-list/ds"
+)
+
+func GetTaskHandler(db *sql.DB) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		id := r.URL.Query().Get("id")
+		if id == "" {
+			http.Error(w, `{"error":"Не указан идентификатор"}`, http.StatusBadRequest)
+			return
+		}
+
+		query := `SELECT id, date, title, comment, repeat FROM scheduler WHERE id = ?`
+		row := db.QueryRow(query, id)
+
+		var task ds.Task
+		err := row.Scan(&task.ID, &task.Date, &task.Title, &task.Comment, &task.Repeat)
+		if err == sql.ErrNoRows {
```

Comment:
Ошибки лучше так сравнивать
```suggestion
		if errors.Is(err, sql.ErrNoRows) {
```